julia-mode.el

```diff
@@ -48,19 +48,15 @@
 
 (defcustom julia-indent-offset 4
   "Number of spaces per indentation level."
-  :type 'integer
-  :group 'julia)
```

Comment:
Can you please explain why the `:group` was removed?

Reply:
Sure (there's also justification in the commit message). Since the group was declared with `:prefix "julia-"`, any `defcustom` that starts with "julia-" will automatically be in the group. So the `:group` is merely redundant.

Reply:
Thanks, I missed that. I didn't know about this feature of `defcustom`. Is this documented?

Reply:
Doesn't look like it. The manual is the only place I see `:prefix` mentioned, and it doesn't talk about implicitly including variables having the prefix.

https://www.gnu.org/software/emacs/manual/html_node/elisp/Group-Definitions.html

I'm aware of the feature because eglot makes use of it. Quickly checked few of emac's builtin progmodes and perl makes use of it, too. If it's an implementation detail, it's one that's at least relatively widely relied upon.

Reply:
Actually, it has nothing to do with `:prefix`. Bad assumption on my part. The elisp manual says:

>If a defcustom does not specify any :group, the last group defined with defgroup in the same file will be used. This way, most defcustom do not need an explicit :group.

https://www.gnu.org/software/emacs/manual/html_node/elisp/Variable-Definitions.html#Variable-Definitions